tutorial/10_key_features/004_distributed.py

```diff
@@ -1,12 +1,30 @@
 """
 .. _distributed:
 
-Distributed Optimization
-========================
+Easy Parallelization
```

Comment:
You can mention why the multi processes work well with optuna. The reason is that we stored the records in the BaseStorage. How about explaining we use BaseStorage to share the information between multi processes? For example, you can create a new section for explaining BaseStorage idea after the `Create a Study` section.

Reply:
Hmm, I'm not quite sure to highlight that Optuna has `BaseStorage` and utilizes it to enable distributed optimization.
Personally `BaseStorage` and the other `Storage` classes are internal things for readers of this tutorial.

Reply:
Ah, if you mean, we should mention that RDB storage is necessary for distributed optimization, then I'd say it's great.